src/gui/lazyfilelist.h

```diff
@@ -345,19 +351,19 @@ class LazyDirView {
         }
 
         bool EntryAccepted() const {
-            if ((fno.fattrib & (AM_SYS | AM_HID)) != 0) {
-                return false; // system and hidden files/directories are not accepted
-            }
-            if ((fno.fattrib & AM_DIR) != 0) {
+            //            if ((fno->d_type & (AM_SYS | AM_HID)) != 0) {
+            //                return false; // system and hidden files/directories are not accepted
+            //            }
```

Comment:
Don't keep commented out code.
If we want to ignore system and hidden files, we should update `filesystem_fatfs.c`.
If it's not important at the moment be we want to add this functionality later, add a TODO comment.